packages/wonder-blocks-icon/components/icon.js

```diff
@@ -44,7 +44,7 @@ const StyledSVG = addStyle("svg");
  * <Icon
  *     icon={icons.search}
  *     color={Color.white}
- *     size={"medium"}
+ *     size="medium"
```

Comment:
I tried to find a eslint rule for this but I couldn't.  I'm kind of surprised that prettier isn't opinionated about this.